leaf-ui/js/displayAnalysis.js

```diff
@@ -31,7 +40,11 @@ function displayAnalysis(analysisResults){
     // ElementList = analysisResults.elementList;
 
     // Update history log
-    updateHistory(currentAnalysis);
+    // updateHistory(currentAnalysis);
+
+    // Update Analysis Config bar
```

Comment:
What is meant to go in this empty space? Or was this comment supposed to be deleted?

Reply:
Supposed to be deleted, thanks for the catch!